pennylane/transforms/decompositions/single_qubit_unitary.py

```diff
@@ -145,9 +141,9 @@ def xyx_decomposition(U, wire, return_global_phase=False):
     ...               [ 0.53396245-0.10177564j,  0.76279558-0.35024096j]])
     >>> decomp = xyx_decomposition(U, 0, return_global_phase=True)
     >>> decomp
-    [RX(array(0.45246584), wires=[0]),
-    RY(array(1.39749741), wires=[0]),
-    RX(array(-1.72101925), wires=[0]),
+    [RX(tensor(-1.72101925, requires_grad=True), wires=[0]),
```

Comment:
Why the angle are different than previously?

Reply:
Notice that the order is just flipped. The angles are the same. Previously the gates were returned in matrix multiplication order (so multiplying left to right gives the unitary). Now, I changed it so that the gates are returned in the order they must be applied to the qubit (in other words, the order in which they should appear in the circuit). So multiplying right to left gives the unitary now.

Reply:
Thank you it is a very good catch!